### PR TITLE
Feature develop login/signup

### PR DESCRIPTION
**Pull request by Quantum-35**
**What does this branch do**
create a welcome page with Login and Signup modals floated to the right

**Description of look to be completed**
User should be able to be redirected to the profile page that matches its role..e.g users to users page, 
and drivers to Drivers page.

**What PT stories?**
[#158452727] https://www.pivotaltracker.com/n/projects/2179783
**Home Page**
homepage

**Signup Model page**
homepage-signup
**Signin Model Page**
homepage-signin

### CODE REVIEW

**Comment by kwanj-k** [comment]
Cool UI

**Comment by kwameasiago** [comment]
Your design is awesome

**Comment by Xerrex** [comment]
I like your UI. Looks intuitive.

**Comment by evamaina** [comment]
very nice UI

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Ride My Way</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./assets/css/style.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="./assets/css/mediaQueries.css" />
+    <!-- <script src="main.js"></script> -->
```
Comment (AnthonyGW) [inline UI/index.html:10]: Remove commented code.

File: UI/index.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+</head>
+<body>
+    <header class='format-header'>
```
Comment (AnthonyGW) [inline UI/index.html:13]: Use consistent quotation marks. Either single marks or double quotes for all the html tags.

**Review by AnthonyGW — CHANGES_REQUESTED** [review]
Add a newline at the end of every file.
Git ignore DS_Store files.
Remove images that are not in use.

**Pull request merged**